pywinauto/timings.py

```diff
@@ -354,38 +414,9 @@ def WaitUntilPasses(
       except TimeoutError as e:
          print("timed out")
          raise e.
-     
-    """
     
-    start = time.time()
+    """    
+    return always_wait_until_passes(timeout, retry_interval, exceptions)(func)(*args)

```

Comment:
I'm not sure. The ```wrapper``` brings args for ```func``` and still needs access to its own parameters from the outer scope